app/controllers/checkouts_controller.rb

```diff
@@ -0,0 +1,95 @@
+class CheckoutsController < ApplicationController
+  rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized, except: :update
+  protect_from_forgery except: :update
+  before_action :store_location
+
+  def checkout
+    authorize :checkout
+  end
+
+  def process_payment
+    authorize :checkout
+
+    # get payment method, if blank, it's credit card, if not, it's bank_slip
+    param          = (params[:token].empty? ? "method" : "token").to_sym
+    payment_method = params[:method] == '' ? 'credit_card' : 'bank_slip'
```

Comment:
When credit card is select on Iugu form, the "method" attribute is empty, but when bank_slip so this have an value. Don't ask why they did this way...